sections/dashboard/Stake/StakingPortfolio.tsx

```diff
@@ -112,9 +112,6 @@ const StakingPortfolio: FC<StakingPortfolioProps> = ({ setCurrentTab }) => {
 };
 
 const StyledTabButton = styled(TabButton)`
-	p {
-		font-size: 12px;
-	}
 	margin-bottom: 9px;
```

Comment:
Yeah. I can add that to the checklist. Another thought I had was it might make sense to add a `margin` prop to some components that frequently need margins, but in every other obscure case, we can just use a spacer.

Reply:
yea also margin prop works :thumbsup: